.travis.yml

```diff
@@ -1,8 +1,11 @@
-dist: trusty
+dist: xenial
```

Comment:
The `xenial` upgrade is needed since we want to use an `nginx` version that supports HTTP/2 proxying.